### PR TITLE
Remove eg and ie

### PR DESCRIPTION
**Pull request by decentralgabe**
Fixes https://github.com/w3c/vc-data-model/issues/1025

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1045.html" title="Last updated on Feb 15, 2023, 6:08 PM UTC (8d12632)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1045/fe54b80...8d12632.html" title="Last updated on Feb 15, 2023, 6:08 PM UTC (8d12632)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
I am not native English, so it is difficult for me to judge this. But what is wrong with "i.e.," or "e.g.,"? Or is it a difference between British and American English (I myself learned British English, originally.)

**Comment by decentralgabe** [comment]
> I am not native English, so it is difficult for me to judge this. But what is wrong with "i.e.," or "e.g.,"? Or is it a difference between British and American English (I myself learned British English, originally.)

Seems like an internationalization issue https://github.com/w3c/vc-data-model/pull/1023#discussion_r1093301954

**Comment by TallTed** [comment]
Yes, this was originally raised (elsewhere) by an i18n review.

File: index.html [hunk 1]
```diff
@@ -3465,11 +3465,11 @@ <h3>Spectrum of Privacy</h3>
             src="diagrams/privacy-spectrum.svg" alt="Horizontal bar with
             red on the left, orange in the middle, and green on the
             right.  The red has the text 'Highly correlatable (global
-            IDs), e.g., government ID, shipping address, credit card
+            IDs), such as: government ID, shipping address, credit card
```
Comment (TallTed) [inline index.html:3468]:
```suggestion
            IDs), such as government ID, shipping address, credit card
```

File: index.html [hunk 2]
```diff
@@ -3465,11 +3465,11 @@ <h3>Spectrum of Privacy</h3>
...
             number'.  The orange has the text 'Correlatable via collusion
-            (personally identifiable info), e.g., name, birthday, zip
+            (personally identifiable info), such as: name, birthday, zip
```
Comment (TallTed) [inline index.html:3470]:
```suggestion
            (personally identifiable info), such as name, birthday, zip
```

File: index.html [hunk 3]
```diff
@@ -3465,11 +3465,11 @@ <h3>Spectrum of Privacy</h3>
...
             code'.  The green has the text 'Non-correlatable
-            (pseudonyms), e.g., age over 21'.">
+            (pseudonyms), such as: age over 21'.">
```
Comment (TallTed) [inline index.html:3472]:
```suggestion
            (pseudonyms), such as age over 21'.">
```

**Review by TallTed — CHANGES_REQUESTED** [review]
colons are generally overkill here, breaking the flow more than needed

**Review by iherman — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by peacekeeper** [comment]
I asked a professional English language editor in my family...

> We're working on a technical specification document, and someone said we should avoid "e.g." and "i.e.", and instead write "such as" and "that is". As a professional editor, do you agree with this?

Her reply:

> No, you can write both ways, especially on technical documents you can use e.g., and i.e., with commas

Me again:

> They say "e.g.," is difficult for some non-native English readers, and therefore for an international audience it should be avoided.

Her again:

> Disagree, but let them do what they want :)

**Comment by TallTed** [comment]
If we decide to keep them (which will require standing up to at least some i18n reviewers), I will note she also advised, `with commas`. :-)

**Comment by msporny** [comment]
@TallTed wrote:
> which will require standing up to at least some i18n reviewers

Hey @r12a, @bert-github, @fantasai, @frivoal, @himorin, @xfq,

The perennial question of whether or not W3C specifications should use latin abbreviations like "e.g.," or if it is preferable to write "for example" instead (from an internationalization standpoint). The last time I checked with some of you, the advice was to always prefer language that required less proficiency in English over language that used latin abbreviations (greater proficiency in English). 

Is that still the current guidance for W3C specifications?

**Comment by iherman** [comment]
> The last time I checked with some of you, the advice was to always prefer language that required less proficiency in English over language that used latin abbreviations (greater proficiency in English).

I am really surprised by all this. Although it was a very long time ago ☹️ I still remember that when I started to learn English (to be specific, essentially British English because we were in Europe) the "e.g." and "i.e." abbreviations were introduced very early. It was part of the basics.

**Comment by decentralgabe** [comment]
Any objections to merging this as is?

**Review by Sakurann — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Comment by aphillips** [comment]
@msporny et al regarding: https://github.com/w3c/vc-data-model/pull/1045#issuecomment-1437594686

The I18N WG is somewhat surprised by this thread. We are not aware of our having--now or in the past--a policy against (mostly Latin derived) abbreviated terms common to English such as `i.e.`, `e.g.`, `cf.`, `etc.` and so forth. If one or another of us made this comment in the past, this is to say: we didn't mean it/sorry.

I do note that some writers or organizations have style guides that prefer spelled out English terms (for clarity in English), but I18N does not take a position on this. I double-checked the [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/) and it is silent on this topic, so I suppose it is not a W3C policy either.

**Comment by decentralgabe** [comment]
Thanks @aphillips I'm open to closing this. I will note there are other places in the spec (outside these changes) that have already removed latin usages. For consistency we'll want to revert them too - if we go down the path of keeping latin.

**Comment by TallTed** [comment]
> For consistency we'll want to revert them

It's great that we now have a clear statement from the I18n WG, and we can keep existin `e.g.` or `i.e.` *if* the meaning of the use is clear, which is not always the case.

That said, I don't see a necessity to use `e.g.` or `i.e.` everywhere they *could* be used, nor to revert every instance we've changed to an appropriate English phrase to the relevant Latin abbreviation.  This kind of consistency is not the same as consistent use of relevant technical vocabulary.

**Comment by brentzundel** [comment]
sounds like this can be closed, yes?

**Comment by decentralgabe** [comment]
Closing as no longer needed.

**Pull request closed without merging**